### PR TITLE
MQE-2140: [chrome81] dragAndDrop with x and y offsets no longer works

### PR DESCRIPTION
**Pull request by soumyau**
### Fixed Issues (if relevant)

1. magento/magento2-functional-testing-framework#<issue_number>: Issue title
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/verification tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)
 - [ ] Changes to Framework doesn't have backward incompatible changes for tests or have related Pull Request with fixes to tests

### CODE REVIEW

**Comment by KevinBKozan** [comment]
> 2. Please check with @KevinBKozan on why we need to use custom steps instead of the web driver drag and drop function in the first place. I remember it's some page builder requirements.
@jilu1 it was a pagebuilder requirement, yes. If you remove them and pagebuilder tests still pass then it should be fine.

File: src/Magento/FunctionalTestingFramework/Module/MagentoWebDriver.php [hunk 1]
```diff
@@ -745,23 +745,24 @@ public function _before(TestInterface $test)
...
-            $action->moveByOffset($travelX, $travelY)->perform();
+            $action->moveToElement($snodes);
+            $action->clickAndHold($snodes);
+            $action->moveByOffset($travelX, $travelY);
             $action->release()->perform();
         } else {
-            parent::dragAndDrop($source, $target);
+            $action->clickAndHold($snodes);
+            $action->moveToElement($tnodes);
+            $action->release($tnodes)->perform();
```
Comment (jilu1) [inline src/Magento/FunctionalTestingFramework/Module/MagentoWebDriver.php:765]: Any reason for this change? I remember this issue only applies for dragAndDrop with offset.

Comment (soumyau) [reply src/Magento/FunctionalTestingFramework/Module/MagentoWebDriver.php:765]: for chrome 81, separating out dragAndDrop into it's native functions helped resolve couple of issues with drag and drop without offsets too, hence the fix.

**Comment by soumyau** [comment]
> 1. Changes look good. I only have one question.
> 2. Please rebase your branch. There are too many revision of the MagentoWebDriver.php in your branch.
Will do 👍 
> 3. From the test fix, it looks like instead of calculating x and y offset, we now use -1. Can we document how user can calculate this value? Will it always be -1?
It's won't be always one, there are tests which use calculated offsets.
> 4. Pull latest mainline and run builds with all required extensions to make sure that fix works in all dragAndDrop tests for Chrome65 and Chrome81 if you have not done so.
I'll trigger another build and add link here 👍

File: src/Magento/FunctionalTestingFramework/Module/MagentoWebDriver.php (hunk 1 above)
Comment (jilu1) [reply src/Magento/FunctionalTestingFramework/Module/MagentoWebDriver.php:765]: Interesting to know. Maybe php webdriver need to make the some change?

Comment (soumyau) [reply src/Magento/FunctionalTestingFramework/Module/MagentoWebDriver.php:765]: yes, we are on latest version of php webdriver. It's probably page builder specific.

**Pull request merged**